### ISSUE TITLE
[recommended.ts] no-reference-import should be specified in rules, not jsRules

### ISSUE THREAD

**Issue by bumbleblym** [palantir/tslint#2440]
### Bug Report

- __TSLint version__: `5.0.0`
- __TypeScript version__: `2.2.2`
- __Running TSLint via__: (pick one) CLI ~~/ Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc~~

#### Actual behavior

```
[0] Following rules specified in configuration couldn't be applied to .js or .jsx files:
[0]     no-reference-import
[0] Make sure to exclude them from "jsRules" section of your tslint.json.
```

#### Expected behavior
`no-reference-import` should be specified in `rules`, not `jsRules`.

[`noReferenceImportRule.ts`](https://github.com/palantir/tslint/blob/b9444e31dfa2557aec5bcd47d71435f71007b5ff/src/rules/noReferenceImportRule.ts#L31)
```typescript
public static metadata: Lint.IRuleMetadata = {
  ...
  typescriptOnly: true,
  ...
}
```
[`recommended.ts`](https://github.com/palantir/tslint/blob/b9444e31dfa2557aec5bcd47d71435f71007b5ff/src/configs/recommended.ts#L251)
```typescript
export const jsRules = {
  ...
  "no-reference-import": true,
  ...
}
```
